client/me/security-2fa-backup-codes/index.jsx

```diff
@@ -147,7 +145,7 @@ class Security2faBackupCodes extends React.Component {
 						disabled={ this.state.generatingCodes || !! this.state.backupCodes.length }
 						onClick={ this.handleGenerateButtonClick }
 					>
-						{ this.props.translate( 'Generate New Backup Codes' ) }
+						{ this.props.translate( 'Generate new backup codes' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.